src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+.calendar {
+  display: flex;
+  padding: $padding;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: $size * $day-count + ($gap * ($day-count - 1)) + (2 * $padding);
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  box-sizing: border-box;
+
+  &__day {
+    box-sizing: border-box;
+    position: relative;
+    display: flex;
+    width: $size;
+    height: $size;
+    border: 1px solid #000;
+    background-color: #eee;
+    cursor: pointer;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: background-color 0.5s, transform 0.5s;
+    }
+
+    @for $day-month from 1 through 31 {
+      &:nth-child(#{$day-month})::before {
+        content: '#{$day-month}';
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+      }
+    }
+  }
+
+  @for $days from 28 through 31 {
+    &--month-length-#{$days} {
+      .calendar__day:nth-child(n + #{$days + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  $day-numder: 'mon' 0,
+    'tue' 1,
+    'wed' 2,
+    'thu' 3,
+    'fri' 4,
+    'sat' 5,
+    'sun' 6;
```

Comment:
Move it to variables file
```suggestion
  $day-numder: 
    'mon' 0,
    'tue' 1,
    'wed' 2,
    'thu' 3,
    'fri' 4,
    'sat' 5,
    'sun' 6;
```

Reply:
![image](https://github.com/mate-academy/layout_calendar/assets/154975449/595d2ee4-8ff5-41a7-aa08-feeacfff015e)